cmd/main.go

```diff
@@ -195,7 +196,8 @@ func main() {
 	for _, providerurl := range RemoteProviderURLs {
 		parsedURL, err := url.Parse(providerurl)
 		if err != nil {
-			logrus.Error(providerurl, "is invalid url skipping provider")
+			er := errors.New(providerurl + "is invalid url skipping provider")
```

Comment:
Might want to make this a meshkit error from `github.com/layer5io/meshkit/errors`
because https://docs.meshery.io/project/contributing/contributing-error#warning

Reply:
Nice. I don’t know that @haanhvu has been introduced to MeshKit Errors, so this is a good note.

Reply:
yes, @gr455 errors.New(..) short des, long des, cause, remedy... .
@haanhvu Should create a new error in errors.go file and use this error in logger.

Reply:
Yep, just like Uzair said. You can refer other places in the codebase like [this one in mesheryctl](https://github.com/meshery/meshery/tree/master/mesheryctl/internal/cli/root) to get the structure of the errors.go file. All the best!

Reply:
@haanhvu  Remember that throwing non-meshkit errors in meshkit logger will cause panic. Thank you @gr455  for noticing, I totally missed this.

Reply:
yes let me see. Thanks you guys for the info.

Reply:
Hey guys, about the new error, I'll create it with `meshkit/errors`.

However, there are existing errors that are not meshkit errors. Should I keep them handled by `logrus`? For example, [this](https://github.com/meshery/meshery/pull/5603/files#diff-c444f711e9191b53952edb65bfd8c644419fc7695c62611dc0fb304b4fb197d6R87-R89).